src/hooks/useSearchManager.js

```diff
@@ -31,27 +30,21 @@ export const useSearchManager = () => {
 
   /* Common */
   const resetPage = () => {
-    delete search.p
+    search.offset = 0
+
     setSearch({ ...search })
   }
 
   const updatePage = (newPage) => {
-    if (newPage === 1) {
-      delete search.p
-    } else {
-      search.p = newPage
-    }
+    search.offset =
+      (newPage - 1) * (search.limit || Number(commonQueries.limit))
```

Comment:
Wouldn't it be safe to assume here that search.limit will always be defined? If so should get rid of the fallback because that number should be defined in the API response.

Reply:
Now `limit` is removed from `commonQueries` 👍